.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+    "semi": true,
+    "trailingComma": "all",
+    "singleQuote": true,
+    "tabWidth": 4
+}
```

Comment:
I think most of our project was using code length 140.
let add `printWidth: 140`